src/rules/noRestrictedGlobalsRule.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license
+ * Copyright 2014 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+
+import * as Lint from "../index";
+
+export class Rule extends Lint.Rules.TypedRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        ruleName: "no-restricted-globals",
+        description: "Disallow specific global variables.",
+        descriptionDetails: Lint.Utils.dedent`
+            Disallowing usage of specific global variables can be useful if you want to allow
+            a set of global variables by enabling an environment, but still want to disallow
+            some of those. For instance, early Internet Explorer versions exposed the current
+            DOM event as a global variable event, but using this variable has been considered
```

Comment:
Nit: escape the "event" in "global variable event" with code to make it a little more clear.

```text
global variable \`event\`, but using
```